.github/workflows/rust.yml

```diff
@@ -40,23 +39,19 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
-      - uses: actions-rs/toolchain@v1
+      - uses: ./.github/actions/setup
         with:
-          profile: minimal
-          toolchain: stable
-          override: true
+          key: actix-web
```

Comment:
the rest of the jobs seem to invoke clippy and include it as a `component` input.  this job does not include it as a toolchain component but still executes clippy,

is the clippy component 
- needed here
- not needed anywhere
- fine as is (not used here, but used elsewhere)

Reply:
That's interesting, according to the [rustup book](https://rust-lang.github.io/rustup/concepts/profiles.html) the minimal profile should not contain clippy. I would add clippy in here, simply so we can be sure that it still works tomorrow.

Reply:
> The minimal profile includes as few components as possible to get a working compiler (rustc, rust-std, and cargo).

> The default profile includes all of components in the minimal profile, and adds rust-docs, rustfmt, and clippy.

So, only cargo {build,test,check} should work with the minimal profile.